specs/protocol/holocene/exec-engine.md

```diff
@@ -21,6 +29,10 @@ After the Holocene hardfork's activation, the L2 block header's `withdrawalsRoot
 insertions and deletions have been applied to the trie. In other words, the storage root should be the same root
 that is returned by `eth_getProof` at the given block number.
 
```

Comment:
A good way to make this very clear is saying that its the storage root corresponding to the post state, exactly how the post global state root is included in the block header

Reply:
added this on at the end of the current description.